app/services/hyrax/statistics/collections/over_time.rb

```diff
@@ -7,7 +7,7 @@ class OverTime < Statistics::OverTime
 
         def relation
           AbstractTypeRelation
-            .new(allowable_types: [Hyrax.config.collection_class])
+            .new(allowable_types: [::Collection, Hyrax.config.collection_class].uniq)
```

Comment:
Since we have a configuration that identifies the collection class, it seems like long term, we would want to remove all references to `::Collection`.  Not doing this requires apps to have a `::Collection` class even if they are using a different class for collections.

It sounds like the problem you are trying to solve is the need to have solr docs that have `has_model_ssim:Collection` return along with other collection results even if the app has migrated to using a different collection class.  I wonder if there is a better way to do that other than forcing an app to have a `::Collection` class.

The simple solution would be a recommendation that if a site wants to include finding `has_model_ssim:Collection` with other collections, then define a class named `::Collection` in the app.  Otherwise, it is not required.

Reply:
is there any chance we could move this forward? 

i agree that in the long term removing `::Collection` references is desirable. however, there are already `::Collection` references in the code base, and i don't think we should block adding statistics support for `Hyrax::PcdmCollection` pending their removal.